pkg/handler/nodes/scalestrategy/safequick.go

```diff
@@ -0,0 +1,21 @@
+package scalestrategy
+
+type SafeQuick struct {
```

Comment:
`scalestrategy.SafeQuick` doesn't express what it is, what it does or how it works. Could it be maybe `scalestrategy.Staircase` or `scalestrategy.Steps` or `scalestrategy.Stepper`?

Reply:
good point, renamed